### DISCUSSION TITLE
The ability to see how many times you use a d/k/g

### DISCUSSION THREAD

**Discussion by Elisee619** [MKT-Playbook/Hub#35] [Feature requests]
You should be able to see how many times you use a certain driver, kart, or glider. It may be useful when comparing multiple different ones, and you can see how good they are for a tour.

**Comment by TitusPosiadly** [comment]
Hey sorry for the late reply. Was very busy the last week.

Are you talking about the number of times a card is calculated to be the best card for a track? In the card list, this would be the sorting option "By Best-On". And I am guessing you would like to have this number only for tracks in the current tour?

**Comment by Elisee619** [comment]
That is correct. I also apologize for my extremely late response, my emails
have been flooded and I just finally started to go through them.

**Comment by TitusPosiadly** [comment]
Yeah, alright. I will figure out a way to implement this. Thanks for the idea.

**Comment by TitusPosiadly** [comment]
Ok, I added a new sort option and some better labeling to the coverage list. Will be coming to the Beta shortly. Let me know if this is what you were imagining.

| Image 1 | Image 2 | Image 3 |
| --- | --- | --- |
![IMG_5056](https://github.com/MKT-Playbook/Hub/assets/79757836/b3612836-f01b-43d8-b955-5562521ea737) | ![IMG_5057](https://github.com/MKT-Playbook/Hub/assets/79757836/5076cae8-7d43-461b-8bda-301f2a8eecee) | ![IMG_5059](https://github.com/MKT-Playbook/Hub/assets/79757836/202f884d-9d05-44ba-86d6-178d999bad14)
